CHANGELOG.md

```diff
@@ -4,6 +4,61 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.com).
 
+## [1.21.4](https://github.com/go-gitea/gitea/releases/tag/1.21.4) - 2024-01-16
+
+* SECURITY
+  * Update github.com/cloudflare/circl (#28789) (#28790)
+  * Require token for GET subscription endpoint (#28765) (#28768)
+* API
+  * Fix `GetCommitStatuses` (#28787) (#28804)
+  * Forbid removing the last admin user (#28337) (#28793)
+  * Fix schedule tasks bugs (#28691) (#28780)
+  * Fix issue dependencies (#27736) (#28776)
+  * Fix: system webhooks API bug (#28531) (#28666)
+  * Refactor CORS handler (#28587) (#28611)
```

Comment:
It is not for "API", but it is a following PR for `  * Fix 405 method not allowed CORS / OIDC (#28583) (#28586)` to fix the CORS bugs.

Reply:
updarted 970ce304b31f48d09402d8af53f9e4883fda3fef